opba-protocols/sandboxes/xs2a-sandbox/src/main/java/de/adorsys/opba/protocol/xs2a/testsandbox/internal/SandboxApp.java

```diff
@@ -64,13 +64,13 @@
 @Getter
 public enum SandboxApp {
 
-    ONLINE_BANKING_UI("adorsys/xs2a-online-banking-ui:2.9", true), // adorsys/xs2a-online-banking-ui
+    ONLINE_BANKING_UI("adorsys/xs2a-online-banking-ui:2.9", true, false), // adorsys/xs2a-online-banking-ui
```

Comment:
Why not simply changing all paths like
`String resourceName = "sandbox/application-" + testProfileName() + (useOldProperties ? "-old" : "") + ".yml";`
to
`String resourceName = "sandbox-old/application-" + testProfileName() + "`yml";`

We do not need 'old' flag here, all resources can be located in sandbox-old folder and it can be deliberately used for BasicTest

Reply:
ok